src/game_api/entities_chars.hpp

```diff
@@ -102,11 +102,11 @@ class Player : public PowerupCapable
     int32_t i168;
     int32_t i16c;
     float y_pos; // not sure why, seams to be the same as abs_y
-    uint8_t jump_flags_a;
+    uint8_t jump_lock_timer;
     /// can jump while airborne if greater than 0
     int8_t coyote_timer;
-    uint8_t jump_flags_c;
-    uint8_t jump_flags_d;
+    uint8_t hired_hand_name;
+    uint8_t swim_timer;
```

Comment:
Some comment's maybe? like `hired_hand_name` is that a stringid? those usually come as 32bit value